share/wake/lib/system/environment.wake

```diff
@@ -31,8 +31,10 @@ publish path = match (getenv "WAKE_PATH")
   None = Nil
 
 # Setup a default path
-publish path = "/usr/bin", Nil
-publish path = "/bin", Nil
+publish path = match (getenv "WAKE_STRICT_PATH")
+  Some _ = Nil
+  None = "/usr/bin", "/bin", Nil
```

Comment:
The way this is written, any value that this is set to will be treated as "true", I think including the empty string. Since it's kind of annoying to actually unset environment variables on the command line (I'm not sure if you can do it in a single state for a single command), I think it'd be better to pattern match on a single "truthy" value and treat all other values as "falsey".

```suggestion
publish path = match (getenv "WAKE_STRICT_PATH")
  Some "1" = Nil
  Some _ = "/usr/bin", "/bin", Nil
  None = "/usr/bin", "/bin", Nil
```